module/platform/webots/Webots/src/Webots.cpp

```diff
@@ -242,13 +260,16 @@ namespace module::platform::webots {
             log<NUClear::FATAL>("Connection was closed.");
         }
 
+        // TODO(KipHamiltons): Does this need to be replaced with our custom clock's now()?
         connect_time = NUClear::clock::now();
```

Comment:
I should clarify, this is currently the real connect time.

```suggestion
        // Set the real time of the connection initiation
        connect_time = NUClear::clock::now();
        // Reset the simulation connection time
        Clock::current_tick = 0;
```